storio-common/src/test/java/com/pushtorefresh/storio/internal/ChecksTest.java

```diff
@@ -34,9 +34,9 @@ public void checkNotNullNegative() {
     public void checkNotNullExceptionMessage() {
         try {
             checkNotNull(null, "expected message");
-            fail("NullPointerException should be thrown");
+            failBecauseExceptionWasNotThrown(NullPointerException.class);
```

Comment:
Sorry, just ignore this comment, looks like I mixed some lines of code in my brain while reviewing, everything is okay